reana.yaml

```diff
@@ -1,11 +1,29 @@
-# TODO
-#
-# command sequence:
-#
-# source /cvmfs/lhcb.cern.ch/lib/LbLogin.sh
-# lb-run DaVinci/latest bash -f
-# root Optimise.C
-# root ModelFixing.C
-# mkdir fitresults
-# mkdir mass_fits
-# root OSMassFit.C 
+version: 0.3.0
+inputs:
+  files:
+    - code/Optimise.C
+    - code/ModelFixing.C
+    - code/OSMassFit.C
+    - code/RooFitHeaders.h
+    - code/lhcbStyle.C
```

Comment:
We need to add also `data/D2PiMuMuOS.root` as input file so that it would be uploaded to the work space